client_code/atomic/utils.py

```diff
@@ -0,0 +1,19 @@
+# SPDX-License-Identifier: MIT
+# Copyright (c) 2021 anvilistas
+
+__version__ = "0.0.1"
+
+
+def get_atom_prop_repr(atom, prop):
+    tp_name = type(atom).__name__
+    if isinstance(atom, dict):
+        return f"{tp_name}[{prop!r}]"
+    return f"{tp_name}.{prop}"
+
+
+class _A:
```

Comment:
a docstring might be worthwhile here. It's not immediately obvious what this is doing.